src/test/multi_stress_watch.cc

```diff
@@ -170,6 +170,7 @@ int main(int args, char **argv)
   else if (type == "ec")
     test_erasure(cluster, pool_name, obj_name);
 
+  cluster.shutdown();
```

Comment:
`Rados` 's destructor calls `Rados::shutdown()`. so this is not necessary.

Reply:
Thanks for reminding.